include/nbl/core/string/stringutil.h

```diff
@@ -11,6 +11,7 @@
 #include <cctype>
 #include "stddef.h"
 #include "string.h"
+#include "utf8.h"
```

Comment:
remove

Reply:
after you remove, you can merge to your branch with the latest stuff (danylo and LoD system)